### PR TITLE
Lobby view cleanup

### PR DESCRIPTION
**Pull request by stevengoldberg**
Refactor lobbyView with BEM and ES6 class, fix webpack source maps

### CODE REVIEW

File: src/js/util.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+export function buildElementClasses (block, elements) {
```
Comment (andrewljohnson) [inline src/js/util.js:7]: this method deserves a comment methinks

File: webpack.config.js [hunk 2]
```diff
@@ -27,6 +25,7 @@ module.exports = {
   resolve: {
     extensions: ['', '.js']
   },
+  devtool: '#source-map',
```
Comment (andrewljohnson) [inline webpack.config.js:28]: what's this?

**Comment by andrewljohnson** [comment]
lgtm besides the comment

File: webpack.config.js (hunk 2 above)
Comment (stevengoldberg) [inline webpack.config.js:28]: https://babeljs.io/docs/usage/options/

**Pull request merged**